Makefile

```diff
@@ -95,13 +95,31 @@ build/psysh-php54-compat: $(PSYSH_SRC) $(PSYSH_SRC_FILES)
 build/%/psysh: vendor/bin/box build/%
 	vendor/bin/box compile --working-dir $(dir $@)
 
+# Signatures
+
+build/%/psysh.asc: build/%/psysh
```

Comment:
and how you deal with people who want to build it for testing purposes?

Reply:
I'm not an expert in `Makefile`. do you know how could this be done to not require signing every time?

Currently, the signature is only required when making `dist`. When making `build`, only PHARs are made.

Reply:
Ah I see, maybe that's fine then

Reply:
Yeah, this will only run for the `dist` target, so it’s where you’d put it.